Algorithms/Ruby/caesar-cipher.rb

```diff
@@ -0,0 +1,21 @@
+def caesar_cipher(str, shift)
+  result = ''
+  str.chars.each do |char|
+    ord = char.ord
+    if ord >= 65 && ord <= 90
```

Comment:
I would make constants instead of using seemingly random numbers in all of these if statements.